metabrainz/templates/oauth/prompt.html

```diff
@@ -16,7 +16,7 @@ <h1 class="page-title">{{ client['name'] }}</h1>
           <img src="{{ url_for('static', filename='img/oauth/identity.svg') }}"/>
         </div>
         <div class="description">
-          Your identity on MetaBrainz
+          {{ _('Your identity on') }} MetaBrainz
```

Comment:
Don't leave out words. Don't make assumptions about how people are going to translate strings.